addon/versions.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+if (Ember.libraries) {
+  Ember.libraries.register('FastBoot', 'FASTBOOT_VERSION');
```

Comment:
if we do this, we should mimic how ED does this:

* ember-data/version module is added
* that module is imported to populate https://github.com/emberjs/data/blob/c9d8212c857ca78218ad98d11621819b38dba98f/addon/-private/core.js#L26